Build failed on master(b93bd661693d7)

```
make
make -C cmd/ipfs-cluster-service build
make[1]: Entering directory '/home/shu/go/src/github.com/ipfs/ipfs-cluster/cmd/ipfs-cluster-service'
go build "-trimpath" -mod=readonly -ldflags "-X main.commit=b93bd661693d7ecc3c1e05a1614f5096dde0fc01"
go: downloading github.com/ipfs/go-mfs v0.1.3-0.20210507195338-96fbfa122164
../../adder/ipfsadd/add.go:21:2: github.com/ipfs/go-mfs@v0.1.3-0.20210507195338-96fbfa122164: invalid version: unknown revision 96fbfa122164
make[1]: *** [Makefile:9: ipfs-cluster-service] Error 1
make[1]: Leaving directory '/home/shu/go/src/github.com/ipfs/ipfs-cluster/cmd/ipfs-cluster-service'
make: *** [Makefile:19: build] Error 2
```

--- a/go.mod
+++ b/go.mod
@@ -37,7 +37,7 @@ require (
 	github.com/ipfs/go-ipns v0.3.0
 	github.com/ipfs/go-log/v2 v2.5.1
 	github.com/ipfs/go-merkledag v0.8.1
-	github.com/ipfs/go-mfs v0.1.3-0.20210507195338-96fbfa122164
+	github.com/ipfs/go-mfs v0.2.1
 	github.com/ipfs/go-path v0.3.0
 	github.com/ipfs/go-unixfs v0.4.1
 	github.com/ipld/go-car v0.5.0

